palantir-java-format-spi/src/main/java/com/palantir/javaformat/java/FormatterService.java

```diff
@@ -48,4 +48,15 @@ ImmutableList<Replacement> getFormatReplacements(String input, Collection<Range<
      *     Style Guide - 3.3.3 Import ordering and spacing</a>
      */
     String formatSourceReflowStringsAndFixImports(String input) throws FormatterException;
+
+    /**
+     * Fixes imports (eg. ordering, spacing, and removal of unused import statements).
+     *
+     * @param input the input string
+     * @return the output string
+     * @throws FormatterException if the input string cannot be parsed
+     * @see <a href="https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing">Google Java
+     *     Style Guide - 3.3.3 Import ordering and spacing</a>
+     */
```

Comment:
I don't think the javadoc adds much - especially as we don't necessarily use Google's import ordering guidelines.